config/initializers/feature_toggle.rb

```diff
@@ -4,4 +4,10 @@ def self.supported_housing_schemes_enabled?
 
     false
   end
+
+  def self.startdate_two_week_validation_enabled?
+    return true if Rails.env.production?
```

Comment:
Can probably just make this true everywhere for now?

Reply:
Ah, I understood that we only wanted it to be true for production but could toggle it off when needed

Reply:
Will update this

Reply:
@baarkerlounger set it to true all the time